src/Microsoft.CodeQuality.Analyzers/Core/Maintainability/DoNotIgnoreMethodResults.cs

```diff
@@ -197,6 +218,33 @@ public override void Initialize(AnalysisContext analysisContext)
                             opContext.ReportDiagnostic(diagnostic);
                         }
                     }, OperationKind.ExpressionStatement);
+
+                    osContext.RegisterOperationAction(opContext =>
```

Comment:
In this operation we want to focus on inner invocations as the other operation only focuses on outer ones.